docs/systemd.md

```diff
@@ -6,18 +6,19 @@ Below is a sample unit file.
 
 ```
 [Unit]
- Description=The Kolide Launcher
- After=network.service syslog.service
-
- [Service]
- ExecStart= $LauncherPath \
- --hostname=$FleetServer:FleetPort \
- --enroll_secret=$FleetSecret \
- --autoupdate \
---osqueryd_path=$OsquerydPath
+Description=The Kolide Launcher
+After=network.service syslog.service
+
+[Service]
+ExecStart= $LauncherPath \
+  --hostname=$FleetServer:FleetPort \
+  --enroll_secret=$FleetSecret \
+  --autoupdate=true \
```

Comment:
This doesn't need `=true` why add it?

Reply:
I initially thought it was the cause of an error I was getting when trying to enable the service so added this. The main objective in this update was the messy spacing though, I just left this in because I personally prefer explicit to implicit.

Reply:
I admire the desire for explicitness. I'm leery, since I think the history of option parsing has often had boolean options throw errors when supplied with an argument. Golang supports this, but it feels very modern to me. 

I'm happy to just accept the spacing cleanup. I can try to decide whether to accept this as is, or push for it to remain.

Reply:
Totally understood. I have no problems with removing the `=true` and just keeping the space updates, if desired.

Reply:
Yes, do that please.

Reply:
Done